### ISSUE TITLE
The Cards examples should not contain implicit headers

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4577]
[Clarity Version]
3.1.1

[Test URL]
https://clarity.design/documentation/cards

[WCAG 2.1 Guideline]
Success Criterion 1.3.1 Info and Relationships
(Level A)

[OS]
any

[User Agents/Assistive Technologies]
any

[Component]
Cards

[Description]
The Cards examples should not contain implicit headers

See the relevant HTML/CSS below.

<div _ngcontent-clarity-c239="" class="card-header"> Header </div>

.card-header, .card-title {
    color: #000;
    color: var(--clr-card-title-color,#000);
    font-weight: 200;
    font-weight: var(--clr-card-title-font-weight,200);
    font-size: .9rem;
    letter-spacing: normal;
}

[User Impact]
Screen reader users will not know that this is a header and what heading level it represents.

[Steps to Reproduce]
- Go to the Test URL
- Read the card header with a screen reader and observe that it is not announced as a header

[Recommendation]
Implement all text visually styled like a header with the appropriate h1 - h6 tag

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
